.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request 💡
+about: Suggest a new idea for the project.
+title: ''
+labels: enhancement
+assignees: ''
+---
+
+[comment]: <> (If the feature happens in multiple SDK, open it in the integration-guide instead: https://github.com/meilisearch/integration-guides/issues)
+[comment]: <> (This is not an exhaustive model but a help. Nothing is mandatory)
+
+**Description**
+Brief explanation of the feature.
+
+**Basic example**
+If the proposal involves something new or a change, include a basic example. Omit this section if it's not applicable.
+
+**Additional context**
+Add any other context about the feature here.
```

Comment:
I don't know if this step is really useful. For me, all the description will be done in the paragraphs above

Reply:
I am shared between not to propose too much and also left space for free expression.
We can change it for:
**Other**
Any other things you want to add.
For now, I remove these paragraphs!

Reply:
I link this "Other", it's more friendly :)

Reply:
Done